tests/br_table_partition/run.sh

```diff
@@ -19,11 +19,14 @@ TABLE="usertable"
 TABLE_COUNT=16
 PATH="tests/$TEST_NAME:bin:$PATH"
 
-run_sql "set @@global.tidb_enable_table_partition ='nightly'"
+# `tidb_enable_list_partition` currently only support session level variable, so we must put it in the create table sql
+SESSION_VARIABLE=
+run_sql "set @@global.tidb_enable_table_partition ='nightly'" || SESSION_VARIABLE="set @@session.tidb_enable_list_partition = 'ON'; "
```

Comment:
this is only used inside prepare.sh... so it seems better this definition be placed just inside prepare.sh

Reply:
done